webapi/src/test/scala/org/knora/webapi/messages/admin/responder/valueObjects/ProjectsValueObjectsADMSpec.scala

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright © 2021 Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.messages.admin.responder.valueObjects
+
+import org.knora.webapi.UnitSpec
+import org.knora.webapi.exceptions.BadRequestException
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectsErrorMessagesADM._
+import org.knora.webapi.messages.store.triplestoremessages.StringLiteralV2
+import zio.prelude.Validation
+
+/**
+ * This spec is used to test the [[ProjectsValueObjectsADM]] value objects creation.
+ */
+class ProjectsValueObjectsADMSpec extends UnitSpec(ValueObjectsADMSpec.config) {
+  "ProjectIRI value object" when {
+//    TODO: check string formatter project iri validation because passing just "http://rdfh.ch/projects/@@@@@@" works
```

Comment:
Do you want to resolve this in this PR? If not, I would delete it and create a JIRA issue instead (if you haven't already). Otherwise I would resolve it in this PR and then get rid of the TODO.
In any case, I think this check is important and should be done.

Reply:
It's possible bug => new bug :)